oss_scripts/pip_package/setup.py

```diff
@@ -73,8 +73,9 @@ def finalize_options(self):
     cmdclass={'install': InstallPlatlib},
     distclass=BinaryDistribution,
     install_requires=[
-        'tensorflow>=2.7.0, <2.8',
-        'tensorflow_hub>=0.8.0',
+        "tensorflow>=2.7.0, <2.8; platform_machine != 'arm64'",
+        "tensorflow-macos>=2.6.0, <2.7; platform_system == 'Darwin' and platform_machine == 'arm64'",
```

Comment:
Do you want to remove the tensorflow-macos requirement? I think it would be better since right now TF has to be built from source to include our master branch's requirements. This way, pip won't reinstall TF with a bad version when this is built.